packages/session-replay-browser/src/helpers.ts

```diff
@@ -1,13 +1,102 @@
 import { getGlobalScope } from '@amplitude/analytics-client-common';
-import { UNMASK_TEXT_CLASS } from './constants';
+import { MASK_TEXT_CLASS, UNMASK_TEXT_CLASS } from './constants';
+import { DEFAULT_MASK_LEVEL, MaskLevel, PrivacyConfig } from './config/types';
+import { getInputType } from '@amplitude/rrweb-snapshot';
 
-export const maskInputFn = (text: string, element: HTMLElement) => {
-  if (element.classList?.contains(UNMASK_TEXT_CLASS)) {
-    return text;
+// compiler guard
+/* istanbul ignore next */
```

Comment:
can we add a todo here and else where? SDK code does have the general standard of 100% test coverage (recognizing that's not a perfect measure, and it's sometimes a pain to upkeep), but would prefer not to start a pattern of exceptions.